### ISSUE TITLE
Request: Could this window be detachable and re-sizable?

### ISSUE THREAD

**Issue by ImSpecial** [cheat-engine/cheat-engine#1454]
I would love to have this area to be with a click of a button or whatever, be detachable and re-sizable in it's own window, any chance this could be added to the list of "I'll think about it" as a possible future enhancement?

Area in question: https://i.imgur.com/skwQOLo.png

Thanks!

**Comment by cheat-engine** [comment]
```
alf=createForm()
alf.BorderStyle=bsSizeable
AddressList.Parent=alf
```

**Comment by ImSpecial** [comment]
I assume that is the solution to my request, unfortunately I've spent the better part of my life as a mechanic then later an EMT, so what you speak is lost on me. : P

I can search for health values though!

**Comment by cheat-engine** [comment]
ctrl+alt+L , paste the code in there, click execute.  Let me know if you need more help with this as I can make it into an autorun script

**Comment by ImSpecial** [comment]
Hey thanks! I was able to figure it out now, and even able to piece together an autorun script as you say by using some of your homework found within (don't worry I changed it a little bit so the teacher won't know!)

I still think a "detach" button for this is a good idea though, so I won't need to remember to backup this script, and so others can also benefit that are not aware, but I can understand not wanting to add a trillions of different buttons to clutter the main GUI too.

Guess I'll close this as it's kinda resolved, and also not to take away from all the "Can't Open" and "Badlion" issues... (Badlion is some kind of meme at this point... right???), but don't forget this topic for one future day when you're bored enough to maybe do it!

Thanks again Dark Byte.
